docker/package/model.py

```diff
@@ -574,3 +577,13 @@ def gen_rules(self, out):
         rules_contents = gen_systemd_rules_contents(self)
         with open(out, 'w') as f:
             f.write(rules_contents)
+
+    def gen_install(self, out):
+        startup_scripts = \
+            list(set(filter(lambda x: x is not None, map(lambda x: x.startup_script, self.systemd_units))))
```

Comment:
a nitpick, but the same spelled out with a list comprehension is both shorter and more idiomatic python: 

```
set([x.startup_script for x in self.systemd_units if x is not None])
```